cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -351,6 +352,72 @@ def _update_postgres_password(self):
         )
         logger.notice('postgres password successfully updated')
 
+    def _create_db_monitoring_account(self):
+
+        def create_user_sql(credentials):
+            delimiter = '$password$'
```

Comment:
It might be best to factor this (password string generation logic) out into a separate function since it's now used here and one other place in this file?